src/test/javascript/spec/component/programming-exercise/feedback-analysis/modals/feedback-filter-modal.component.spec.ts

```diff
@@ -47,11 +49,13 @@ describe('FeedbackFilterModalComponent', () => {
         const closeSpy = jest.spyOn(activeModal, 'close');
 
         component.filters.occurrence = [component.minCount(), component.maxCount()];
+        component.filters.errorCategories = ['Student Error', 'Ares Error'];
         component.applyFilter();
 
         expect(storeSpy).toHaveBeenCalledWith(component.FILTER_TASKS_KEY, []);
         expect(storeSpy).toHaveBeenCalledWith(component.FILTER_TEST_CASES_KEY, []);
         expect(storeSpy).toHaveBeenCalledWith(component.FILTER_OCCURRENCE_KEY, [0, 10]);
+        expect(storeSpy).toHaveBeenCalledWith(component.FILTER_ERROR_CATEGORIES_KEY, ['Student Error', 'Ares Error']);
```

Comment:
Can we be more specific here? (also for the calls before)

```suggestion
        expect(storeSpy).toHaveBeenNthCalledWith(4, component.FILTER_ERROR_CATEGORIES_KEY, ['Student Error', 'Ares Error']);
```

Reply:
not really as we call them with different values before applying the filter and after applying the filter